src/styles/utils/variables.scss

```
@@ -0,0 +1,16 @@
+$number-positions:
+  1 60px 370px,
+  2 145px 445px,
+  3 250px 470px,
+  4 350px 443px,
+  5 430px 365px,
+  6 465px 244px,
+  7 430px 120px,
+  8 350px 45px,
+  9 250px 20px,
+  10 145px 40px,
+  11 60px 115px,
+  12 17px 238px;
```

cl-3 (Positioning in pixels is a bad idea, I'll explain why: because if your clock is for example 600px or 700px, then your numbers will float away) they will be incorrectly positioned. Therefore, it is better to use percentages for positioning.)
```suggestion
$number-positions:
  1 13% 75%,
  2 29% 88%,
  3 47% 93%,
  4 69% 88%,
  5 83% 75%,
  6 93% 49%,
  7 430px 120px,
  8 350px 45px,
  9 250px 20px,
  10 145px 40px,
  11 60px 115px,
  12 3% 77%;
```
I didn't fix everything, but the idea is to use percentages.